tasks/queue-granules/index.js

```diff
@@ -71,6 +87,7 @@ function updateGranuleBatchCreatedAt(granuleBatch, createdAt) {
  *   that is passed to the next task in the workflow
  **/
 async function queueGranules(event, testMocks = {}) {
+  let deconstructedId = [];
```

Comment:
do you mind just moving this to being before line 115? The reason is that it just doesn't need to be declared in this scope and, technically, pmap runs concurrently so it would be possible for one loop of pmap to update it before another loop is done using it, though I think that's unlikely in it's current form.